src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -41,6 +39,108 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
             _symbolSearchService = symbolSearchService;
         }
 
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
this was a general move of this function.  Though some of hte logic around options/services got rejiggered.